kcbq-connector/src/test/java/com/wepay/kafka/connect/bigquery/integration/utils/BucketClearer.java

```diff
@@ -0,0 +1,38 @@
+package com.wepay.kafka.connect.bigquery.integration.utils;
+
+/*
+ * Copyright 2016 WePay, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+
+import com.google.cloud.storage.Storage;
+import com.wepay.kafka.connect.bigquery.GCSBuilder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class BucketClearer {
```

Comment:
Do you think it is useful to add a javadoc here?

Reply:
Sure, added.